src/view-components/widget/widget.test.tsx

```diff
@@ -80,4 +89,21 @@ describe(`<Widget />`, () => {
     expect(element).not.toBeNull();
     done();
   });
+
+  it(`should get the SingleSpaContext`, async done => {
+    const { queryByText, debug } = render(
+      <Widget
+        widgetConfig={{
+          name: "test",
+          esModule: "@openmrs/esm-patient-chart-widgets",
+          usesSingleSpaContext: true
+        }}
+      />
+    );
+
+    const element = await waitForElement(() => queryByText("Test Widget"));
+    expect(element).not.toBeNull();
```

Comment:
Is there a way to check that the context has indeed been injected? something like:
```
expect(element.prop. singleSpaContext.mountParcel).not.toBeNull();
```
This will ensure that the spa context will always be passed when people do modifications to the widget component.

Reply:
I can't figure out how to get props use react testing library. The way we could test this would be to see that if the element returned had the prop "singleSpaContext" with value of {mountParcel: function}. @fatmali do you know to test for props?